src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -327,12 +327,17 @@
 
 helps['functionapp config ssl import'] = """
 type: command
-short-summary: Import an SSL certificate to a function app from Key Vault.
+short-summary: Import a certificate to a functionapp. The certificate can be from Key Vault, or an App Service Certificate.
 examples:
-  - name: Import an SSL certificate to a function app from Key Vault.
+  - name: Import an SSL certificate to a function app from Key Vault. Option '--key-vault-certificate-name' has been deprecated, use '--certificate-name' instead
```

Comment:
the word SSL certificate was removed above but here we are still using that. Any reason for this difference?

Reply:
@singlessis can you run az webapp config ssl import --help & share how the results look with your updates?